zipnn/util_safetensors.py

```diff
@@ -30,7 +30,7 @@ def build_compressed_tensor_info(uncompressed_tensor: torch.tensor) -> Compresse
     returns metadata to be saved for the respective compressed tensor.
     """
     return CompressedTensorInfo(
-        dtype=str(uncompressed_tensor.dtype).removeprefix("torch."),
+        dtype = str(uncompressed_tensor.dtype).replace("torch.", ""),
```

Comment:
```suggestion
        dtype = str(uncompressed_tensor.dtype)
        if dtype.startswith("torch."):
                dtype = dtype[len("torch."):]
```

so that it will replace just the first occurrence.